### PR TITLE
Libcloud-based Azure Adapter.

### PR DESCRIPTION
**Pull request by maugustosilva**
Initially, no support for volume attachments (the libcloud backend does not allow it) or for
vmcapture (ditto).

### CODE REVIEW

File: configs/templates/_as.txt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+CLOUD_SERVICE_PREFIX = $AS_CLOUD_SERVICE_PREFIX
+KEY_NAME = $AS_KEY_NAME
+SSH_KEY_NAME = $AS_SSH_KEY_NAME
+ACCESS = $AS_ACCESS
+SECURITY_GROUPS = not_yet_applicable
+
+[AI_TEMPLATES : HADOOP ]
+HADOOP_HOME = /usr/local/hadoop
+
+# DigitalOcean has made public images available for the following 3 workloads:
```
[the remark this answers is not in the document]
Comment (maugustosilva) [reply configs/templates/_as.txt]: done

File: configs/templates/_as.txt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+SIZE = from_vm_template
+LEAVE_INSTANCE_ON_FAILURE = $False
+FORCE_FAILURE = $False
+DISABLE_TIMESYNC = $False
+USERDATA = $True
+USERDATA_SSH = $True
+IMAGE_PREFIX = $EMPTY
+IMAGE_SUFFIX = $EMPTY
+
+# We're not as big as amazon yet. Go easy on us please.
```
[the remark this answers is not in the document]
Comment (maugustosilva) [reply configs/templates/_as.txt]: done

File: configs/templates/_as.txt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+ATTACH_PARALLELISM = 1
+CAPTURE_SUPPORTED = $True
+RUNSTATE_SUPPORTED = $True
+RESIZE_SUPPORTED = $True
+LOGIN = $AS_LOGIN
+SSH_KEY_NAME = $AS_SSH_KEY_NAME
+CREDENTIALS = $AS_CREDENTIALS
+DONT_START_QEMU_SCRAPER = $True
+
+# We're not as big as amazon yet. Go easy on us please.
```
[the remark this answers is not in the document]
Comment (maugustosilva) [reply configs/templates/_as.txt]: done

File: configs/templates/_as.txt [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+ACCESS = $AS_ACCESS
+SECURITY_GROUPS = not_yet_applicable
+
+[AI_TEMPLATES : HADOOP ]
+HADOOP_HOME = /usr/local/hadoop
+
+# DigitalOcean has made public images available for the following 3 workloads:
+[VM_TEMPLATES : AS_CLOUDCONFIG ]
+CHECK = size:ExtraSmall, imageids:1, imageid1:to_replace
+TINYVM = size:ExtraSmall, imageids:1, imageid1:cloudbench-nullworkload-on-1604.060117-1
```
[the remark this answers is not in the document]
Comment (maugustosilva) [reply configs/templates/_as.txt]: done

**Comment by ibmcb** [comment]
I have updated this PR as requested. Please let me know if this is ok now. Thanks.

**Pull request merged**